corehq/apps/userreports/templatetags/__init__.py

```diff
@@ -0,0 +1 @@
+# This file is intentionally left empty to mark this directory as a Python package.
```

Comment:
nit: was this file added by an AI code generator?
Just wondering since we usually don't add this line.